### ISSUE TITLE
Auto-TLS not working

### ISSUE THREAD

**Issue by Joniator** [pterodactyl/panel#3722]
### Is there an existing issue for this?

- [X] I have searched the existing issues before opening this issue.

### Current Behavior

Auto-TLS is currently not working on a fresh ubuntu install.

`sudo wings --debug --auto-tls --tls-hostname example.com` logs "webserver is now listening with auto-TLS enabled; certificates will be automatically generated by Let's Encrypt hostname=gsg1.tazed.club".  But I never see a log messagethat the certificate was aquired, the logs look fine (See below). But the daemon is not recognized by the panel.

`curl https://localhost:8080` returns
curl: (35) error:14094438:SSL routines:ssl3_read_bytes:tlsv1 alert internal error.

The configuration is automatically loaded from the instance with `wings configure --node`

As soon as I install certbot, generate a cert on my own for the same domain I put in auto-tls, everything works fine.

### Expected Behavior

I start wings with --auto-tls --tls-hostname example.com, and wings generates a certificate for me and the demon uses the certificate.

Side note: The documentation does not mention auto-tls at all, maybe this could be adressed to ;)

### Steps to Reproduce

Install wings exactly like the documentation says, and start the demon with auto-tls:
`/usr/local/bin/wings --auto-tls --tls-hostname example.com`

### Panel Version

1.6.3

### Wings Version

1.5.3

### Error Logs

```bash
https://ptero.co/ocexuseten

The log contains at least one start without the --auto-tls Flag (The stacktrace), the others are with auto-tls and the correct tls-hostname.
```

**Comment by PerennialWheat** [comment]
> curl https://localhost:8080
> ```
did you try `curl https://yourdomain.com:8080`? The certificate you requested is not for localhost it is for your domain (ex: https://yourdomain.com)

**Comment by parkervcp** [comment]
this should be resolved by #3707

**Comment by Joniator** [comment]
> this should be resolved by #3707
I don't use the compose-file, and this is related to wings. My panel is working just fine with other wings, just not when they use auto-tls 😄 

> > curl https://localhost:8080
> 
> did you try `curl https://yourdomain.com:8080`? The certificate you requested is not for localhost it is for your domain (ex: https://yourdomain.com)

I did not, you are right. 
I don't know why I did not see this log entry, it seems to show up rather sporadically, and far from every time: I start the daemon: acme/autocert: unable to satisfy "https://acme-v02.api.letsencrypt.org/acme/authz-v3/45564867400" for domain "gso1.example.com": no viable challenge type found

Which was very likely caused by my firewall, which I changed on one server (the one I also changed to certbot that startet working), because I can also no longer replicate the issue
